src/mame/layout/j2lucky2.lay

```diff
@@ -206,7 +206,7 @@
 		<rect state="0">
 			<color red="0.25" green="0.25" blue="0.00"/>
 		</rect>
-		<text string="When avaliable after ">
+		<text string="When available after ">
```

Comment:
Please provide a photo of the machine showing that this is not an original spelling error.